examples/framework.rs

```diff
@@ -108,6 +108,7 @@ async fn setup<E: Example>(title: &str) -> Setup {
 
     log::info!("Initializing the surface...");
 
+    #[cfg(not(webgl))]
     let backend = if let Ok(backend) = std::env::var("WGPU_BACKEND") {
```

Comment:
we should still allow `BROWSER_WEBGPU` being picked if compiling for wasm

Reply:
webgl alias is not wasm, it is wasm PLUS webgl feature, so if compiled for wasm without webgl feature, WebGPU will be allowed.

Reply:
Ideally I think we'd like to allow both WebGPU and WebGL (as a fallback) to be used in a single wasm file, instead of only WebGL or WebGPU

Reply:
Outch, it seems that Wgpu-rs has not been designed to permit this …
backend/mod.rs is clearly a toggle between WebGPU backend and others, so making a build enabling WebGPU AND another backend is kind of a big rework …

Reply:
Oh right, I forgot about that. We can consider doing that later then, but let's just be sure to write the feature checks in a way that would still allow it